domain/consensus/utils/txscript/standard_test.go

```diff
@@ -28,14 +28,14 @@ func mustParseShortForm(script string, version uint16) []byte {
 	return s
 }
 
-// newAddressPubKeyHash returns a new util.AddressPubKeyHash from the
-// provided hash. It panics if an error occurs. This is only used in the tests
+// newAddressPubKey returns a new util.AddressPubKey from the
+// provided public key. It panics if an error occurs. This is only used in the tests
 // as a helper since the only way it can fail is if there is an error in the
 // test source code.
-func newAddressPubKeyHash(pkHash []byte) util.Address {
-	addr, err := util.NewAddressPubKeyHash(pkHash, util.Bech32PrefixKaspa)
+func newAddressPubKey(publicKey []byte) util.Address {
```

Comment:
Please rename this function to NewAddressPublicKey and the type AddressPubKey to AddressPublicKey.